content/updates/governance/2-Flexible-product-models-variant-products.md

```diff
@@ -0,0 +1,13 @@
+# Product models and variants enhanced flexibility
+
+## Convert variant products into simple products
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/enrich-products-variants.html#convert-a-variant-product-to-a-simple-one"
+
+To offer greater flexibility when dealing with variant products, you can now easily transform them into simple products directly from the PIM interface. Quite a game changer when you need to refine your family variants! You can also use it if your product range now contains one product instead of several.
+
+On a side note, this new feature will also help you change the family variants of your product model more efficiently. A real time saver indeed.
+
+## Update the variant axis value of your product models
```

Comment:
```suggestion
## Update the variant axis value of your products
```
If I am correct, it's for PM if they are 2 levels of variations and for any variant, so I'd rather remove model from the title, it could be misleading.

Reply:
Even cleaner and clearer:
```suggestion
## Update the axis value of your variant products
```